AdminUi/src/AdminUi/Program.cs

```diff
@@ -111,6 +111,7 @@ static void Configure(WebApplication app)
             .AddDefaultSecurityHeaders()
             .AddCustomHeader("Strict-Transport-Security", "max-age=5184000; includeSubDomains")
             .AddCustomHeader("X-Frame-Options", "Deny")
+            .AddCustomHeader("Access-Control-Allow-Credentials", "true")
```

Comment:
Why is this necessary? As far as I understand, it is only required if you want to send your credentials to **other** sites. But since our UI is delivered by the API-server, this is not the case.

Reply:
We need to review the proper way to deploy the AdminUi, maybe write a README with instructions for development. In the past few days, I have not been able to get Angular to run when launching the AdminUi in Visual Studio. When I did, however, the API and the UI were still being served from different ports, hence that special header.

Reply:
Okay, but after deployment it is just a single server. So you should only add this header if necessary. You can for example add a correpsonding boolean property to the CorsConfiguration that is then stringified for to pass it to "AddCustomHeader".